.circleci/config.yml

```diff
@@ -12,25 +12,42 @@ jobs:
       - run:
           name: Install Dependencies
           command: npm install
-      - run:
-          name: Unit Tests
-          command: npm test -- auth/
       - run:
           name: Build Application
-          command: yarn ng build --prod --aot=false --build-optimizer=false --no-progress
+          command: yarn ng build --prod --aot=false --build-optimizer=false
       - persist_to_workspace: 
-          root: dist
+          root: ./
           paths: 
             - ./
   test:
     <<: *defaults
     steps:  
       - attach_workspace : 
           at: ~/clark-client
+      - run:
+          name: Clone e2eBox Repository 
+          command: >-
+           ls && cd .. && git clone https://${github_username}:${github_password}@github.com/nvisal1/CLARK-e2eBox.git
+      - run:
+          name: Install Docker client
+          command: |
+            set -x
+            VER="17.03.0-ce"
+            curl -L -o /tmp/docker-$VER.tgz https://download.docker.com/linux/static/stable/x86_64/docker-$VER.tgz
+            tar -xz -C /tmp -f /tmp/docker-$VER.tgz
+            mv /tmp/docker/* /usr/bin
+      - run: 
+          name: Install Docker Compose
```

Comment:
This will need to happen before setting up remote docker engine

https://circleci.com/docs/2.0/docker-compose/